### DISCUSSION TITLE
Unable to launch

### DISCUSSION THREAD

**Discussion by aastha51551** [JdeRobot/RoboticsAcademy#2875] [Q&A]
09:39:16 [MainThread  ] [INFO ] (my_app_logger)  Starting RAM consumer in <manager.comms.websocker_server.WebsocketServer object at 0x7f6c01310160>:7163
INFO:manager.comms.websocker_server:Listening on port 7163 for clients..
INFO:manager.comms.websocker_server:Starting WebsocketServer on thread Thread-1 (serve_forever).

the terminal freezes after this and when i try to open http://127.0.0.1:7164/ i get :
This page isn’t working
127.0.0.1 didn’t send any data.

**Comment by jmplaza** [comment]
Hi @aastha51551 , please follow the updated developer documentation of RoboticsAcademy! [https://github.com/JdeRobot/RoboticsAcademy/blob/humble-devel/README.md](https://github.com/JdeRobot/RoboticsAcademy/blob/humble-devel/README.md)
